pandas/core/nanops.py

```diff
@@ -373,7 +383,16 @@ def nanvar(values, axis=None, skipna=True, ddof=1):
     # but NOT going to coerce them to the Timedelta type
     # as this could cause overflow
     # so var cannot be computed (but std can!)
-    return _nanvar(values, axis=axis, skipna=skipna, ddof=ddof)
+
+    if not skipna:
+        return values.var(axis=axis, ddof=ddof)
+    else:
+        np_nanvar = getattr(np, 'nanvar', None)  # Present in Numpy >= 1.8
+        if np_nanvar is None:
+            return _nanvar(values, axis=axis, skipna=skipna, ddof=ddof)
```

Comment:
I would rather put ``_nanvar`` here. Unless we are calling it for another reason?